### PR TITLE
Allow sessionTokenAction to POST redirect

### PR DESCRIPTION
**Pull request by winzou**
I am currently implementing a PSP (2C2P) that needs a POST redirect from the session capture action.
Currently the session capture action only redirects via GET.
This PR adds the ability to forward a POST request, depending on the incoming request method.

### CODE REVIEW

File: Controller/CaptureController.php [hunk 1]
```diff
@@ -19,12 +22,22 @@ public function doSessionTokenAction(Request $request)
...
             )
-        )));
+        ));
+
+        if ($request->isMethod('GET')) {
+            return $this->redirect($redirectUrl);
+        }
+
+        try {
+            throw new HttpPostRedirect($redirectUrl, $request->request->all());
```
Comment (makasim) [inline Controller/CaptureController.php]: There is a listener that could convert reply to the response. Could we use it?

Comment (winzou) [reply Controller/CaptureController.php]: Done, I didn't know it was listening all controller actions!

**Comment by winzou** [comment]
Great, thanks!
Could you tag a new release please?

**Comment by makasim** [comment]
done

**Pull request merged**